src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.targets

```diff
@@ -8,7 +8,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ResolvedFileToPublish Include="App_Data\**" Exclude="App_Data\**\*.log">
+    <ResolvedFileToPublish Include="App_Data\**" Exclude="$(DefaultItemExcludes);App_Data\**\*.log">
```

Comment:
Why the semicolon?

Reply:
That's the msbuild convention to concatenate items collections

Reply:
Ok